.pre-commit-config.yaml

```diff
@@ -43,6 +43,11 @@ repos:
         entry: python tools/generate_requirements.py
         files: "pyproject.toml|requirements/.*\\.txt|tools/generate_requirements.py"
 
+  - repo: https://github.com/woodruffw/zizmor-pre-commit
+    rev: 4dcc98ccce605605fd6d5202bf803a8c8bc34e8d # v1.1.1
```

Comment:
Any value in already updating to most recent [v1.3.0](https://github.com/woodruffw/zizmor-pre-commit/releases/tag/v1.3.0)?

Reply:
Yes, let me do that.